README.md

```diff
@@ -11,6 +12,59 @@ You can reach the maintainers of this project at:
 - [Slack channel](https://kubernetes.slack.com/messages/sig-node)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
 
+ 
+## Building the operator
+
+Checkout the sources
+
+```bash
+$ git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+
+Build the operator image
+
+```bash
+make image IMAGE=<my repo>:<my tag>
+```
+
+Optionally you can push it to your image repo 
+
+```bash
+make image-push IMAGE=<my repo>:<my tag>
+```
+
+## Manual deploy of the operator
+
+Update the  `Makefile` with the custom image built and configure the namespace where the operator should be deployed.
+
```

Comment:
One last nit: with the current instructions (like `make X IMAGE=<my repo>:<my tag>`) this is not required. You could move this paragraph e.g. under the "Building" section above, as an alternative.

*Alternatively, instead of specifying variables on the command line, you can edit the `Makefile` to permanently change parameter defaults like name of the image or namespace where the operator is deployed.*

WDYT?